src/routes/safe/components/Balances/SendModal/screens/ReviewTx/index.jsx

```diff
@@ -67,7 +68,11 @@ const ReviewTx = ({
 
     if (!isSendingETH) {
       const StandardToken = await getStandardTokenContract()
+      const HumanFriendlyToken = await getHumanFriendlyToken()
```

Comment:
Isn't this a super class of the standard token, if so it should be enough to use this and remove the standard token here